airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -113,19 +113,29 @@ public SQLDialect getSqlDialect() {
 
   private Certificates cachedCerts;
 
-  public synchronized Certificates getCertificates() {
+  public Certificates getCertificates() {
     if (cachedCerts == null) {
-      final String caCert, clientKey, clientCert;
-      try {
-        caCert = getContainer().execInContainer("su", "-c", "cat ca.crt").getStdout().trim();
-        clientKey = getContainer().execInContainer("su", "-c", "cat client.key").getStdout().trim();
-        clientCert = getContainer().execInContainer("su", "-c", "cat client.crt").getStdout().trim();
-      } catch (IOException e) {
-        throw new UncheckedIOException(e);
-      } catch (InterruptedException e) {
-        throw new RuntimeException(e);
+      synchronized(this) {
+        if (cachedCerts == null) {
+          final String caCert, clientKey, clientCert;
+          try {
+            caCert = getContainer().execInContainer("su", "-c", "cat ca.crt").getStdout().trim();
+            getContainer().execInContainer("su", "-c", "openssl ecparam -name prime256v1 -genkey -noout -out client_"+getUserName()+".key");
```

Comment:
Call String.format() or the .formatted method on a string literal instead of these + concatenations please.

Reply:
fair